src/webcomponents/commons/layouts/custom-vertical-navbar.js

```diff
@@ -301,7 +306,11 @@ export default class CustomVerticalNavBar extends LitElement {
                                     <hr class="mt-0 mb-3">
                                     <!-- TODO: CONTENT in a div -->
                                     <div class="settings-content-wrapper" id="settings-content-wrapper">
-                                        ${subItem.render(this.opencgaSession, this.study)}
+                                        ${
+                                        (this.organization) ?
+                                            subItem.render(this.opencgaSession, this.organization) :
+                                            subItem.render(this.opencgaSession, this.study)
+                                        }
```

Comment:
This code was developed some time before the organization object was included in this.opencgaSession. It will be refactored accordingly in the next task